content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -426,12 +524,25 @@ same root of trust.
 
 ## Uninstalling the remote cluster
 
-To uninstall the remote cluster run the following command:
+To uninstall the remote cluster, run the following command:
 
 {{< text bash >}}
-$ istioctl --context=${REMOTE_CLUSTER_CTX} manifest generate <your original remote configuration> | \
-    kubectl delete -f -
+$ istioctl --context=${REMOTE_CLUSTER_CTX} manifest generate -f istio-remote0-cluster.yaml | \
+    kubectl delete -f - --context=${REMOTE_CLUSTER_CTX}
```

Comment:
I think you need to remove the blank lines in the text bash as only yet first line copies and runs.

Reply:
Removing the lines does get the whole block selected.